geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/vector/SimpleFeatureVector.scala

```diff
@@ -245,7 +252,18 @@ object SimpleFeatureVector {
       val isLong = dateVector.exists(_.isInstanceOf[BigIntVector])
       if (isLong) { Encoding.Max } else { Encoding.Min }
     }
-    val encoding = SimpleFeatureEncoding(fidEncoding, geomPrecision, datePrecision)
+    val axisOrderPrecision = {
+      if (java.lang.Boolean.getBoolean("org.geotools.referencing.forceXY") ||
+        Hints.getSystemDefault(Hints.FORCE_LONGITUDE_FIRST_AXIS_ORDER) == java.lang.Boolean.TRUE) {
+        Encoding.Max /* Lon/Lat */
+      } else {
+        CRS.getAxisOrder(sft.getCoordinateReferenceSystem) match {
```

Comment:
we always treat everything as EPSG:4326, so this could probably be hard-coded. I guess the exception could be postgis, again.

Reply:
I'd like to avoid having gt-referencing as a dependency, if possible... just to minimize the surface area of this change.

Reply:
Dependency removed